proto/elys/incentive/apr.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+
+package elys.incentive;
+
+import "gogoproto/gogo.proto";
+import "cosmos_proto/cosmos.proto";
+
+option go_package = "github.com/elys-network/elys/x/incentive/types";
+
+message AprEdenElys {
```

Comment:
Can you add a bit of comment on what this struct does and where it's used?

Reply:
@jelysn 
It is for tracking APR for each reward program.